### PR TITLE
added a check to several JSON editor commands to prevent crashes for css===null

### PR DESCRIPTION
**Pull request by ArnoldSmith86**
I fell for this so often.

We check `typeof variable != 'object'` and forget that `null` is also type `object`.

### CODE REVIEW

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: https://test.virtualtabletop.io/PR-1347/pr-test (or any other room on that server)

After merging, a backup will be available at https://beta.virtualtabletop.io/editor/PR1347-pr-test.

File: client/js/jsonedit.js [hunk 1]
```diff
@@ -283,7 +283,7 @@ const jeCommands = [
     name: 'convert to nested object',
     context: '^.* ↦ css',
     show:  function() {
-      if(typeof jeStateNow.css != "object")
+      if(typeof jeStateNow.css != "object" || jeStateNow === null)
```
Comment (rogerl50) [inline client/js/jsonedit.js:286]: I think you want `jeStateNow.css == null` here.

**Review by rogerl50 — CHANGES_REQUESTED** [review]

**Comment by rogerl50** [comment]
> I fell for this so often.
> 
> We check `typeof variable != 'object'` and forget that `null` is also type `object`.

It also drives me crazy that arrays return type object.

File: client/js/jsonedit.js (hunk 1 above)
Comment (ArnoldSmith86) [reply client/js/jsonedit.js:286]: Oh. I missed `.css`. Duh.

**Review by rogerl50 — APPROVED** [review]
Approved through code review and test.

**Pull request merged**